build.gradle

```diff
@@ -610,7 +610,11 @@ artifacts {
     archives jar
     archives javadocJar
     archives sourcesJar
+
 }
+//remove zip and tar added by the application plugin
```

Comment:
Will our maven upload still have all the artifacts required by maven central? Isn't there a javadoc zip?